buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkDimensions.kt

```diff
@@ -0,0 +1,105 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.skyscanner.backpack.tokens
+
+import com.google.common.base.CaseFormat
+import com.squareup.kotlinpoet.buildCodeBlock
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.MemberName
+import com.squareup.kotlinpoet.PropertySpec
+import com.squareup.kotlinpoet.TypeSpec
+
+interface BpkDimensions : Map<String, Int>
+
+object BpkDimension {
+
+  sealed class Category : BpkParser<BpkDimensions> {
```

Comment:
It looks like there's multiple size token categories that we're not parsing (e.g. typesettings and carousel-indicator-dots) - should we parse these and add some check to fail the task if there are any categories we're not parsing? Even if we don't have components for carousel-indicator-dots the app might need them and we don't miss out on new tokens. typesettings we'll need anyways.

Reply:
There's also lineheight, but that should probably also be treated as sp instead of dp. Maybe we need to update the types at some point to be more explicit.

I'm not necessarily talking about these specific ones right now - just if we actually want to necessarily restrict ourselves to the known types or rather take what comes from foundations as the source of truth. but probably for that we first need to clean up what comes from foundations and revisit that later